tutor/src/tours/teacher-calendar.json

```diff
@@ -27,7 +27,8 @@
   "steps": [
     {
       "title": "Welcome to your dashboard!",
-      "body": "Think of your dashboard as base camp -- you can create and manage assignments, see student progress, and review the book."
+      "body": "Think of your dashboard as base camp -- you can create and manage assignments, see student progress, and review the book.",
+      "customComponent": "CenteredWheel"
     }, {
```

Comment:
a general observation - we need to add automated js linting to travis.  That'll catch things like this.  I haven't because there's too many failures atm.